### DISCUSSION TITLE
USB connection for N64 controller

### DISCUSSION THREAD

**Discussion by upintheairsheep** [sanni/cartreader#964] [Ideas & Suggestions]
Idea: Add a mode where the N64 controller port on the unit translates into a USB HID when plugged into a computer or device. Similar ideas can be implemented in the future with game cartridges from various systems including special hardware, which could be translated to generic USB functionality, like the Game Boy Camera being used as a USB webcam.

**Comment by delaysystem** [comment]
the atmega2560 used in HW5 doesn't have native usb support, also adding usb will add costs due to regulator ics required in the usb2 specification. this means adding usb will require a separate chip and other chips. HW6/HW7 is expected to use Pi Pico and STM32, both with usb support.
